HowToUse.txt

```diff
@@ -1,5 +1,5 @@
 
-How to map randomize your pokemon heart gold or soul silver roms:
+How to map randomize your Pokémon HeartGold or soul SoulSilver:
```

Comment:
There is an extra "soul" here that should be removed.
Furthermore, I think adding a "ROMs" at the end of the sentence makes sense to indicate what is actually being randomized.

New sentence: `How to map randomize your Pokémon HeartGold or SoulSilver ROMs:`